api-docs/list-clientsbyzip.md

```diff
@@ -0,0 +1,17 @@
+#list-clientsbyzip API
+#POST Request
+#URL /api/list-clientsbyzip/
```

Comment:
Instead of a separate url endpoint for this, why not just do this as a query parameter to the `GET /clients` url?

For example: `GET /clients?zip=84109`